packages/core/integrations/tests/doubles/dummy-integration-class.js

```diff
@@ -0,0 +1,90 @@
+const { IntegrationBase } = require('../../integration-base');
+
+class DummyModule {
+    static definition = {
+        getName: () => 'dummy'
+    };
+}
+
+class DummyIntegration extends IntegrationBase {
```

Comment:
What's a test double?

Reply:
A test double is a stand-in object you use in a test in place of a real collaborator. It lets you isolate the system under test, control hard-to-reproduce situations, and make tests fast and deterministic (e.g., no real network, clock, DB).